RELEASE/scripts/sl_ascend.ash

```diff
@@ -2653,7 +2653,13 @@ boolean doBedtime()
 		use(1, $item[resolution: be more adventurous]);
 	}
 
-	if((my_daycount() <= 2) && (freeCrafts() > 0) && !in_tcrs())
+	// If in TCRS skip using freecrafts but alert user of how many they can manually use.
+	if((in_tcrs()) && (freeCrafts() > 0))
+	{
+		print("Using My Free Crafts will have no benefit.", "red");
```

Comment:
Probably worth telling the user that we're skipping free crafts because it's TCRS. How about
```
Skipping end-of-day free crafting since our usual food and drink might be rubbish in TCRS.
Consider manually using your {freeCrafts()} free crafts.
```

Reply:
Good Idea. Done.